### DISCUSSION TITLE
ADS reports retention date, extend the 90 days limit

### DISCUSSION THREAD

**Discussion by mkhalawi** [amzn/ads-advanced-tools-docs#295] [Q&A] [answered]
How can I get Sponsored_Brands and Sponsored_Display reports data with retention date exceeds the 90 days limit? In another world, I need to get ADS data for like a year back. Is that possible in any way? even if it was not reporting! like other API types.

Thanks

**Comment by kdwivedi13** [comment, marked as answer]
Hey Mkhalawi,

Thank you for initiating this thread. As of today, it is not possible, but we are working on this feature. Please continue to engage here and follow the release notes. I would like to request you to create a support ticket and let us know how you intend to use historical data up to 1 year. This will help us prioritize this feature accordingly. When it comes to the closed beta phase, we might include you as well.

Thanks,
Kapil

**Reply by mkhalawi** [reply]
Thank you for your response. I appreciate the efforts your team is putting into developing this feature. Here’s why access to historical ad campaign data up to 12 months is critical for us:
1- Performance Analysis and Optimization: We aim to perform comprehensive trend analysis and performance evaluations over extended periods. This enables us to identify patterns and seasonality, optimize budget allocation, and refine targeting strategies to maximize ROI.
2- Benchmarking and Forecasting: Historical data is essential for benchmarking current campaigns against past performances. It also aids in creating accurate forecasts and setting realistic targets based on year-over-year trends.
3- Campaign Strategy and Planning: Access to a full year of historical data allows us to design more effective campaign strategies. By understanding what has worked well in the past, we can replicate successful tactics and avoid previous pitfalls.
4- User Reporting and Transparency: Our users expect detailed reports and transparency regarding their ad spend and performance. Providing a year-long perspective helps in building trust and demonstrating the long-term value of our strategies.

Having beta access to this feature would significantly enhance our ability to deliver superior services to our clients and drive more value through informed decision-making. We are eager to engage further and provide any additional information or feedback that might help prioritize this feature.

Thank you for considering our request.
